dags/analyzer_helper/telegram/tests/integration/test_telegram_extract_raw_data.py

```
@@ -0,0 +1,421 @@
+                    }
+                ],
+                "interactions": [
+                    {
+                        "name": "reply",
+                        "type": "emitter",
+                        "users_engaged_id": ["4444"],
+                    }
+                ],
+            },
+        )
+        inserted_data = list(self.rawmemberactivities_collection.find())
```

Ruff

Local variable `inserted_data` is assigned to but never used

Remove assignment to unused variable `inserted_data`

(F841)